features/editor_can_edit_articles_written_by_journalists.feature

```diff
@@ -0,0 +1,26 @@
+Feature: Editor can edit articles written by journalists
+
+    As an editor
+    In order to modify an article written by a journalist
+    I would like to be able to edit i
```

Comment:
"I would like to be able to edit i"  You're missing a "T"

Reply:
done!